index.md

```diff
@@ -6,6 +6,25 @@ layout: default
 
 ![](./media/installer_screenshot.png)
 
+## Installation and Getting Started
+
+The pre-requisites for RTVS are: 
+
+* Visual Studio 2015 Community, Professional or Enterprise
+* Visual Studio 2015 Update 3
+* An R interpreter: CRAN-R or Microsoft R Open
+* Note that the Visual Studio 2017 release of RTVS is not quite ready yet, but
+  will ship soon.
```

Comment:
I'd add that products based on VS Integrated Shell (such as Test Professional or SQL Management Studio) lack necessary components and cannot be used with RTVS. The setup now blocks on them.